tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/FmiIntegrationTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.NativeInterop;
+using Microsoft.Identity.Test.Integration.Infrastructure;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.Identity.Test.Unit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Test.Integration.NetCore.HeadlessTests
+{
+    [TestClass]
+    public class FmiIntegrationTests
+    {
+        private const string _fmiAppUrn = "urn:microsoft:identity:fmi";
```

Comment:
style for constants is different I believe. `SomeConstant`

Reply:
We should add an analyzer for it

Reply:
It exists as suggestion today, I agree the UI isn't very clear about it (the 3 little dots0

![image](https://github.com/user-attachments/assets/84af2ade-02f1-48d4-854d-87ab38974c30)

You can make it a warning.

![image](https://github.com/user-attachments/assets/7189f699-ee51-4b57-9886-2add1c387e72)